### PR TITLE
 Fix nav links to work with file:// URLs

### PR DESCRIPTION
**Pull request by pledbrook**
This is an attempt to change the links in the left navigation of the user manual so that they work from file:// URLs. The change seems to work OK, but hasn't yet been tested with the production infrastructure.

Attempts to solve [developer-experience#6](https://github.com/gradle/developer-experience/issues/6).

### CODE REVIEW

File: subprojects/docs/src/main/resources/header.html [hunk 1]
```diff
@@ -88,128 +88,128 @@
             <input type="search" name="q" id="search-input" class="search-input" placeholder="Search the docs"/>
         </div>
         <ul>
-            <li><a href="/userguide/userguide.html">Docs Home</a></li>
+            <li><a href="userguide.html">Docs Home</a></li>
             <li><a href="https://guides.gradle.org">Tutorials</a></li>
             <li><a href="/release-notes.html">Release Notes</a></li>
```
Comment (eriwen) [inline subprojects/docs/src/main/resources/header.html]: :x: seems like we'll want to make this relative, too

**Comment by eriwen** [comment]
I went ahead and made the one little fix and pushed. Thanks again @pledbrook. 

/cc @adammurdoch and others who asked for this... should be in the next nightly.

**Comment by adammurdoch** [comment]
This is great, thank you @pledbrook and @eriwen.

**Comment by eriwen** [comment]
Yeah, sorry it was broken for so long in the first place.

**Pull request merged**